main.go

```diff
@@ -0,0 +1,135 @@
+// Copyright (c) 2020 Doc.ai and/or its affiliates.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"context"
+	"net/url"
+	"os"
+	"time"
+
+	"github.com/networkservicemesh/sdk/pkg/registry/common/setid"
+
+	"github.com/networkservicemesh/api/pkg/api"
+	"google.golang.org/grpc/health"
+	"google.golang.org/grpc/health/grpc_health_v1"
+
+	nested "github.com/antonfisher/nested-logrus-formatter"
+	"github.com/kelseyhightower/envconfig"
+	"github.com/networkservicemesh/api/pkg/api/registry"
+	"github.com/sirupsen/logrus"
+	"github.com/spiffe/go-spiffe/v2/spiffetls/tlsconfig"
+	"github.com/spiffe/go-spiffe/v2/workloadapi"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials"
+
+	"github.com/networkservicemesh/sdk/pkg/registry/core/chain"
+	"github.com/networkservicemesh/sdk/pkg/registry/memory"
+	"github.com/networkservicemesh/sdk/pkg/tools/debug"
+	"github.com/networkservicemesh/sdk/pkg/tools/grpcutils"
+	"github.com/networkservicemesh/sdk/pkg/tools/log"
+	"github.com/networkservicemesh/sdk/pkg/tools/signalctx"
+)
+
+// Config is configuration for cmd-registry-memory
+type Config struct {
+	ListenOn  url.URL `default:"unix:///listen.on.socket" desc:"url to listen on" split_words:"true"`
+	ConnectTo url.URL `default:"unix:///connect.to.socket" desc:"url to connect to" split_words:"true"`
```

Comment:
Why we need ConnectTo?

Reply:
Actually we not need ConnectTo. Removed, thanks!